satellite/gc/observer.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gc
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/bloomfilter"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+)
+
+// Observer implements the observer interface for gc
```

Comment:
This name and comment do little to help me understand what this things does.

Reply:
How is "Observer implements the metainfo loop Observer interface, allowing us to get segment information"

Reply:
great!